com.unity.visualeffectgraph/Editor/Inspector/VisualEffectEditor.cs

```diff
@@ -1333,9 +1333,9 @@ public void OnInspectorGUI()
                     if (m_RenderingLayerMask != null)
                     {
                         string[] layerNames = null;
-                        var srpAsset = QualitySettings.renderPipeline ?? GraphicsSettings.currentRenderPipeline;
+                        var srpAsset = GraphicsSettings.currentRenderPipeline;
```

Comment:
This seems unrelated to the rest of the PR. I'm not up to speed with what the intent is here, so could you expand a bit what's happening? Are we treating `GraphicsSettings.currentRenderPipeline` as the single source of truth as to which RP asset is active? If so, should we make a comprehensive pass over the codebase and remove the remaining uses of `QualitySettings.renderPipeline`? Or are there still valid cases where we want to use the value from QualitySettings?

Reply:
That is related in a sense that I have cleaned the code around my changes :)
And yes there is an upcoming PR (trunk + package) to clean our API changes. 
The idea is that currentRenderPipeline = quality + graphics
but there are 3 APIs so I keep missusing them because we need to clean that